pkg/reconciler/pipelinerun/resources/resultrefresolution.go

```diff
@@ -46,13 +46,19 @@ func ResolveResultRefs(pipelineRunState PipelineRunState, targets PipelineRunSta
 		}
 		allResolvedResultRefs = append(allResolvedResultRefs, resolvedResultRefs...)
 	}
+	return removeDup(allResolvedResultRefs), nil
+}
+
+// ResolvePipelineResultRefs resolves any ResultReference that are found in the target ResolvedPipelineRunTask
```

Comment:
Suggest updating this comment to `ResolvePipelineResultRefs takes a list of PipelineResults and resolves any references they include to Task results in the given PipelineRunState`.

Reply:
Good catch, thank you!